mama/c_cpp/src/gunittest/c/mamamsg/msggeneraltests.cpp

```diff
@@ -960,7 +964,7 @@ TEST_F (MsgGeneralTestsC, DISABLED_msgSetNewBufferInValidBuffer)
     ASSERT_EQ (mamaMsg_setNewBuffer(mMsg, 0, size), MAMA_STATUS_NULL_ARG);
 }
 
-TEST_F (MsgGeneralTestsC, msgSetNewBufferInValidSize)
+TEST_F (MsgGeneralTestsC, DISABLED_msgSetNewBufferInValidSize)
```

Comment:
Complete mistake on my part, that wasn't meant to be in the commit :)
I've re-enabled it now (768241ab37c418a30f109a3d672b5aafd2c03588).